text_extensions_for_pandas/array/tensor.py

```diff
@@ -214,6 +247,12 @@ def __setitem__(self, key: Union[int, np.ndarray], value: Any) -> None:
             raise NotImplementedError(f"__setitem__ with key type '{type(key)}' "
                                       f"not implemented")
 
+    def __eq__(self, other):
+        """
+        Return self==value.
+        """
+        return self == other
```

Comment:
Won't this lead to infinite recursion?

Reply:
I forgot the mixin already defines this and others. It actually overrides this so this particular method is never called. Taking it out.